source/MaterialXTest/MaterialXGenShader/GenShader.cpp

```diff
@@ -141,6 +141,43 @@ TEST_CASE("GenShader: Shader Translation", "[translate]")
     }
 }
 
+TEST_CASE("GenShader: Gltf Transmission Translation", "[translate]")
```

Comment:
For now, I would recommend omitting this new unit test, as what we really want is _render validation_ for a wider set of shader translation use cases, including the new case for which you've added support.

At the moment, we have just one shader translation test in our GitHub Actions suite, as we're currently using CPU renders and the overhead of each new test is quite high, but this is the location where we'll want to add new tests when GPU renders are supported:

https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/.github/workflows/main.yml#L265